src/components/MembersSectionNew/components/MembersCard/index.tsx

```diff
@@ -22,15 +22,15 @@ export default function MembersCard({ member }: { member: MemberType }) {
     reduxDispatch(
       setIsUserRoleUpdateModalVisible({
         visibility: true,
-        username: member.username,
+        username: member.id,
         isUserMember: true,
       })
     );
   }
 
   function openSkillUpdateModal() {
     hideSetting()
-    reduxDispatch(setUserSkillModalVisibility({visibility: true}))
+    reduxDispatch(setUserSkillModalVisibility({visibility: true, username: member?.id, picture: member?.picture?.url, firstName: member?.first_name, lastName: member?.last_name}))
```

Comment:
this can be written more clearly 
```
const skillModalPayload = {
visibility: true,
username: member?.id,
picture: member?.picture?.url, 
firstName: member?.first_name, 
lastName: member?.last_name
}
```
this becomes more readable